src/content/en/2021/page-weight.md

```diff
@@ -1,21 +1,176 @@
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vThwYNAo2K8eVJVNH_4TuA7IZpG534jXn5vFWphLLuHUo_VNGehDMKoIFWf0GquINRKWNHX66MXUmn9/pubchart?oid=356516185&format=interactive",
+  sheets_gid="2090334936",
+  sql_file="request_type_distribution_2021.sql"
+) }}
+
+The request distribution shows that the difference between desktop and mobile is not significant, with desktop leading the way. Something worth noting: the median request on desktop at this time is the same [as last year](../2020/page-weight#page-requests) (74), yet the page weight has ticked up (141 kb), which is consistent with the latest trajectory.
+
+{{ figure_markup(
+  image="requests-content-type.png",
+  caption="Median number of requests per mobile page by content type.",
+  description="Bar chart showing the median number of requests per mobile page by content type. The median number of image requests per page is 23, 20 for JS, 7 for CSS, and 2 for HTML. Desktop and mobile tend to be equal except desktop pages load slightly more image and JS requests.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vThwYNAo2K8eVJVNH_4TuA7IZpG534jXn5vFWphLLuHUo_VNGehDMKoIFWf0GquINRKWNHX66MXUmn9/pubchart?oid=568469823&format=interactive",
+  sheets_gid="2090334936",
+  sql_file="request_type_distribution_2021.sql"
+) }}
+
+Images again make up the largest number of requests, though JavaScript is closing in as the gap has narrowed slightly in the last year.
+
+### File formats
+
+{{ figure_markup(
+  image="response-distribution-format.png",
+  caption="Distribution of image sizes by format.",
+  description="Box plot of the distribution of image sizes by format: gif, ico, jpg, png, svg, and webp. JPG sticks out as having the highest distribution with a 90th percentile exceeding 190 KB per image, up from the previous year. WebP has overtaken PNG as second highest at about 93 KB at the 90th percentile. PNG is now third with smaller 90th percentile than WebP, its 75th percentile is higher. gif, ico, and svg all have relatively tiny distributions.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vThwYNAo2K8eVJVNH_4TuA7IZpG534jXn5vFWphLLuHUo_VNGehDMKoIFWf0GquINRKWNHX66MXUmn9/pubchart?oid=1911661099&format=interactive",
+  sheets_gid="38001790",
+  sql_file="requests_format_distribution.sql"
+) }}
+
+We know that images responsible for a large percentage of web page weight. The above graphic shows the top sources of image weight and the weight distribution. Top 3: JPG, WebP and PNG. So not only is the JPG the most popular image format, it also tends to be the largest by size as well - even larger than a lossless format like the PNG. Likely, the rise in popularity for WebP is due to all major browsers finally supporting the file format. PNG remains popular for use cases such as icons and logos.
```

Comment:
This statement is a little misleading:

> So not only is the JPG the most popular image format, it also tends to be the largest by size as well - even larger than a lossless format like the PNG.

In my head, I see this as a good thing because it might mean that people recognize the limited cases where you might want to use a PNG. Because in most circumstances where you use a JPG, the PNG analog would be larger.